README.md

```diff
@@ -14,29 +14,100 @@ Khiva provides a mean for time series analytics at scale. These analytics can be
-2. Run `conan remote add conan-mpusz https://api.bintray.com/conan/mpusz/conan-mpusz`.
-3. Create `build` folder and, after moving into the new folder, run `conan install .. --build missing`.
-4. Run `cmake ..`. 
-5. Run `make -j8 && make install`.
+## Windows users
+### Installation
+#### Prerequisites
+- Install [Python-64bits](https://www.python.org/downloads) and add the path to the environment variables path. 32-bits version won't work
+- Install [ArrayFire 3.5.1](https://arrayfire.com/download/) and add the path to the environment variables path
+- Install [Vcpkg](https://docs.microsoft.com/es-es/cpp/vcpkg#installation) and add the path to the environment variables path
+- Install [chocolatey](https://chocolatey.org/) to manage windows dependencies and add the path to the environment variables path
 
-### Generating the Khiva installers
-This project can be packaged to be installed in a Windows, Linux or Mac OS system. CPack is used in order to perform this task.
-1. For **Windows** the installer can be generated running the command `cpack -G NSIS` inside the `build` folder. You need [NSIS](http://nsis.sourceforge.net/Download) installed in your system.
-2. For **linux** either a **deb** or a **rpm** package the installer can be generated. This could be done running the command `cpack -G DEB` or `cpack -G RPM` respectively inside the `build` folder.
-3. For **Mac OS** the installer can be generated running the command `cpack -G productbuild` inside the `build` folder.
+#### Process
+- Run `choco install cmake.install -NoNewWindow -Wait` Note: Add the path to the environment variables path and **before** than chocolately environment variable path.
+- Run `choco install doxygen.install -NoNewWindow -Wait`
+- Run `choco install graphviz -NoNewWindow -Wait`
+- Run `python -m pip install --upgrade pip`
+- Run `pip3 install sphinx breathe sphinx_rtd_theme`
+- Run `vcpkg install --triplet x64-windows gtest eigen3 benchmark boost`
+- Create a `build` folder in the root path of the project
+- Browse inside the `build` folder
+- Run `cmake .. -DCMAKE_TOOLCHAIN_FILE="<PATH_TO_VPKG>/scripts/buildsystems/vcpkg.cmake" -DKHIVA_USE_CONAN=OFF -G "Visual Studio 15 2017 Win64"` (Do not forget to clean the build directory everytime before running this command)
+- Run `cmake --build . --config Release -- /m` to compile
 
-**Notes:** Before generating the installer the project should be built. The generated package will be stored in the `build` folder.
+#### Install Khiva library without installer
+As a final step of the installation process:
+- Run `cmake -DBUILD_TYPE=Release -P cmake_install.cmake`
 
-## Generating documentation
+### Generating the Khiva installer
+CPack is used in order to perform this task.
 
+**Notes:** Before generating the installer the project should be built following the process explained in the `Process` section above. The generated package will be stored in the `build` folder.
```

Comment:
**Notes:** Before generating the installer, the project must be built by following the steps in the previous **Process** section. The generated package will be stored in the `build` folder.